testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/UserInfoTest.java

```diff
@@ -246,6 +262,219 @@ public void testSuccess_postMethod_header_textEntity() throws Exception {
         }
     }
 
+
+    @Test
+    public void testSuccessEncryptedAndSignedResponse() throws Exception {
+        testUserInfoSignatureAndEncryption(org.keycloak.crypto.Algorithm.RS256, JWEConstants.RSA_OAEP, JWEConstants.A256GCM);
+    }
+
+    @Test
+    public void testSuccessEncryptedResponseAlgRSA_OAEPEncA256GCM() throws Exception {
+        testUserInfoSignatureAndEncryption(null, JWEConstants.RSA_OAEP, JWEConstants.A256GCM);
+    }
```

Comment:
It seems good to add a test that sets null to "enc" to test a default behavior (expects A128CBC-HS256).

Reply:
Hi @tnorimat , 
about testing various combination of signing and encryption algorithms we have following:

- signing alg: 12 possible algs - 3 HS algs not of interest for our use case (simmetric key not available to clients for signature's check); we'll consider 9 total algs

- encryption alg: 3 possible algs

- encryption enc: 6 possible alg + 1 for default; we'll consider 7 total algs

In tests provided I check all combination of encryption enc and alg (21 tests) and in 18 of them I set also a signing algorithm (each signing alg is used in 2 different test), remaining 3 tests only apply encryption without signing; could be ok?

Reply:
@mposolda Yes, I agree with you.
@giacomoa Could you follow @mposolda 's comment?